ResearchKit/Common/ORKInstructionStep.h

```diff
@@ -63,6 +63,8 @@ ORK_CLASS_AVAILABLE
  */
 @property (nonatomic, copy, nullable) UIImage *image;
 
+@property (nonatomic, copy, nullable) UIImage *auxiliaryImage;
```

Comment:
This is a module developed by Thread that we are using in mPower. The auxiliaryImage is a second image used on some screens. The auxiliary image is overlaid with the main image to show a two-part image where only one part is tinted.

![simulator screen shot aug 16 2016 1 20 55 pm](https://cloud.githubusercontent.com/assets/5649217/17714396/7abb79be-63b4-11e6-991b-e56429225c0b.png)